src/prompt.c

```diff
@@ -574,6 +574,11 @@ prompt_menu(const char *prompt, const struct menu_item *items, int *selected)
 			break;
 
 		default:
+			if (key.modifiers.control && (key.data.bytes[0] == 'C')) {
```

Comment:
If I recall correctly regarding how the keys are encoded you are only supposed to read `data.bytes` if the multibytes flag is set. I think it would be safer to define a macro to match Ctrl-C that check the flags etc.